apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -339,10 +340,13 @@ class NewRewardClass extends React.Component {
     <HorizontalContainer>
       <FormField
         label="Start date"
+        width={`calc(50% - ${GU}px)`}
         required
         input={
           <DateInput
             name="dateStart"
+            horizontal="left"
```

Comment:
Why define a `vertical` prop if it's not going to be used?

Reply:
For future use cases. One issue I keep encountering is that the the DatePicker overflows the element engaging the scrollbar. If instead we allow it to go up instead of down, the user wont have to scroll down to use the DatePicker. But at the moment I though it best to not engage in making those design decision and left the current behaviour as is.

Reply:
I see what you mean. Good call!